.github/workflows/rspec_tests.yaml

```diff
@@ -31,11 +31,5 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.cfg.ruby }}
-
-      - name: Update rubygems and install gems
-        run: |
-          gem update --system --silent --no-document
-          bundle config set without packaging
-          bundle install --jobs 4 --retry 3
-      
+          bundler-cache: true
```

Comment:
Updating rubygems used to be necessary for some or our older rubies. Curious as to why this was dropped and replaced with `bundler-cache`? Could you add comments in the git commit?

Reply:
this keyword is a bit... misleading. It's required to install the gems from the Gemfile and it will do caching. between CI runs.

Reply:
updated the commit with a description